homeassistant/helpers/start.py

```diff
@@ -1,26 +1,28 @@
 """Helpers to help during startup."""
 from __future__ import annotations
 
-from collections.abc import Awaitable, Callable
+from collections.abc import Callable
```

Comment:
You would need this as well...

```suggestion
from collections.abc import Awaitable, Callable
```